test/src/jupyterhub_config/custom_spawner_test.py

```diff
@@ -446,17 +446,19 @@ def test_configure_environment_missing_pythonpath(spawner):
     assert f"{user_env_dir}/lib/python3.11/site-packages" == spawner.environment['PYTHONPATH']
 
 
-@pytest.mark.parametrize("is_admin, expected_dir", [
-    (True, '/cdm_shared_workspace'),  # Admin user case
-    (False, 'to_be_defined_in_the_test')  # Non-admin user case
+@pytest.mark.parametrize("is_admin", [
+    True,  # Admin user case
+    False  # Non-admin user case
 ])
-def test_configure_notebook_dir(is_admin, expected_dir, spawner, caplog):
+def test_configure_notebook_dir(is_admin, spawner, caplog):
     spawner.user.admin = is_admin
     username = 'testuser'
```

Comment:
This would probably be simpler as just a loop at this point, like

```
for b in [True, False]:
    spawner.user.admin = b
    ...
```

Reply:
👍